src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -324,10 +324,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('acr token update') as c:
         c.argument('scope_map_name', options_list=['--scope-map'], help='The name of the scope map associated with the token. If not specified, running this command will disassociate the current scope map related to the token.', required=False)
 
+    for scope in ['acr token create', 'acr token credential generate']:
+        with self.argument_context(scope) as c:
+            c.argument('days', options_list=['--days'], help='Number of days for which the credentials will be valid. If not specified, the expiration will default to the max value "9999-12-31T23:59:59.999999+00:00"', type=int, required=False)
```

Comment:
expiration_in_days? I think there's existing guideline of not putting unit in parameter name.

Reply:
You are right. The existing `az acr token credential generate` already uses the same name, let me update that one as well . Will put in deprecate. Since the whole command is still under preview, so the eventual breaking is fine.